src/app/components/SubjectHeading/LocalLoadingLayer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default props => (
+  <div className="nypl-column-half bibsList subjectHeadingShowLoadingWrapper">
```

Comment:
I know this works. But I think this is the only component using this shorthand and not explicitly naming the component. So, I'd rather follow the same pattern of the other components because this could be confusing to someone later on.

Reply:
Sounds good, changed.